ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -123,6 +154,15 @@ public StatementExecutorResponse execute(
       );
     }
 
+    if (shouldGeneratePlan(executionContext, statement).isPresent()) {
+      return StatementExecutorResponse.handled(Optional.of(
+          new WarningEntity(injected.getStatementText(),
+          "Cannot add `"
+              + shouldGeneratePlan(executionContext, statement).get()
+              + "`: An entity with the same name already exists."
```

Comment:
Could you edit the message so it is similar to the previous message? The below message is thrown in Error or Warnings currently.

```
String.format("Cannot add table %s: A %s with the same name "
                    + "already exists.",
                sourceName, sourceType.toLowerCase()));
```